packages/paste-core/components/tabs/src/Tabs.tsx

```diff
@@ -4,12 +4,14 @@ import {Flex} from '@twilio-paste/flex';
 import {useTabPrimitiveState, TabPrimitiveInitialState} from '@twilio-paste/tabs-primitive';
 import {TabsContext} from './TabsContext';
 
-export type TabsProps = TabPrimitiveInitialState;
+export interface TabsProps extends TabPrimitiveInitialState {
+  variant?: 'fitted' | undefined;
```

Comment:
nit: probably would have been nicer to share these across the various components rather than repeat them 3 times

Reply:
I added a types file and moved `Orientation` and `Variants` types there.